packages/proxy/package.json

```diff
@@ -74,6 +74,7 @@
     "@opentelemetry/resources": "^1.18.1",
     "@opentelemetry/sdk-metrics": "^1.18.1",
     "ai": "2.2.22",
+    "braintrust": "link:../../../sdk/js",
```

Comment:
Will need to set this to the correct SDK version once we land https://github.com/braintrustdata/braintrust-sdk/pull/65.